app/src/main/java/org/simple/clinic/overdue/Appointment.kt

```diff
@@ -382,7 +382,11 @@ data class Appointment(
       FROM Appointment A
       INNER JOIN Patient P ON P.uuid = A.patientUuid
       LEFT JOIN PatientAddress PA ON PA.uuid = P.addressUuid
-      LEFT JOIN PatientPhoneNumber PPN ON PPN.patientUuid = P.uuid AND PPN.deletedAt IS NULL
+      LEFT JOIN (
+        SELECT * FROM PatientPhoneNumber
+        WHERE deletedAt IS NULL
```

Comment:
Is there a reason why we haven't added a condition in the tests to verify this behaviour?

Reply:
We cannot actually add multiple phone numbers to our test nor do we have a public API to directly add the phone number unless we directly inject the DAO object bypassing repository. That's why.